modelcheck/src/main/kotlin/ModelCheck.kt

```diff
@@ -107,24 +129,61 @@ fun writeJunitXml(models: Iterable<SModel>,
                 is IssueKindReportItem.PathObject.ModelPathObject -> {
                     path.resolve(project.repository)!!
                 }
+
                 is IssueKindReportItem.PathObject.NodePathObject -> {
                     val node = path.resolve(project.repository)
                     node.model!!
                 }
+
                 else -> fail("unexpected item type")
             }
         }
 
+
+    // TODO remove these debugging statements
+    allErrors.forEach {
+        printError("Error: " + it.message + " ")
+    }
+
+    errorsPerModule.forEach { entry ->
+        entry.value.forEach {
+            printError("Module error for [${entry.key.moduleName}]: ${it.message}")
+        }
+    }
+
+    errorsPerModel.forEach { entry ->
+        entry.value.forEach {
+            printError("Model error for [${entry.key.name}]: ${it.message}")
+        }
+    }
+
     val xmlMapper = XmlMapper()
 
+    @Suppress("DEPRECATION")
     when (format) {
         ReportFormat.ONE_TEST_PER_MODEL -> {
             val testcases = oneTestCasePerModel(models, errorsPerModel, project)
```

Comment:
Print a deprecation warning here to STDERR and also to the log.
The warning should tell the user what to do ("Use `--format=module`").